test/e2e/specs/specs-playwright/wp-gutenboarding__happy-path.ts

```diff
@@ -1,5 +1,6 @@
 /**
  * @group calypso-pr
```

Comment:
Once we add this test to the `release` group, let's remove it from the `calypos-pr` group and re-enable the skipped test steps. Doesn't have to be in this PR though.

Reply:
Done!